app/views/stations/index.erb

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
we only want this structure of an HTML page on our `layout.erb` - which is always rendered on each page